lidar_localization_node:
  ros__parameters:
    side: 0 # 0 - Blue, 1 - Yellow
    debug_mode: false
    visualize_candidate: true
    likelihood_threshold: 0.01
    consistency_threshold: 0.90
    frame_id: base_footprint
    robot_parent_frame_id: map
